code-challenges.test.js

```diff
@@ -15,6 +15,15 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe(fib, () => {
+    it("returns an array that length containing the numbers of the Fibonacci sequence.") () =>
+  expect(fib(6).toEqual([1, 1, 2, 3, 5, 8])
+
+  describe(fib, () => {
+      it("returns an array that length containing the numbers of the Fibonacci sequence.") () =>
+    expect(fib(10).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+
+
```

Comment:
You are doing better. Here is the structure following the syllabus

describe(fib, () => {
    it("returns an array that length containing the numbers of the Fibonacci sequence.") () =>
  expect(fib(6).toEqual([1, 1, 2, 3, 5, 8])
    expect(fib(10).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
)}
)}